server/app/views/components/Icons.java

```diff
@@ -3,62 +3,106 @@
 import j2html.tags.ContainerTag;
 import services.question.types.QuestionType;
 
-/** Class to hold constants for icons and provide methods for rendering SVG components. */
+/**
+ * Class to hold constants for icons and provide methods for rendering SVG components. Most of these
+ * icons are from https://fonts.google.com/icons, each one is commented with its icon name.
+ */
 public class Icons {
+
+  // Place
   public static final String ADDRESS_SVG_PATH =
       "M12 2C8.13 2 5 5.13 5 9c0 5.25 7 13 7 13s7-7.75 7-13c0-3.87-3.13-7-7-7zm0 9.5c-1.38"
           + " 0-2.5-1.12-2.5-2.5s1.12-2.5 2.5-2.5 2.5 1.12 2.5 2.5-1.12 2.5-2.5 2.5z";
   public static final String ANNOTATION_SVG_PATH =
       "M7 8h10M7 12h4m1 8l-4-4H5a2 2 0 01-2-2V6a2 2 0 012-2h14a2 2 0 012 2v8a2 2 0 01-2 2h-3l-4 4z";
-  // Check
-  public static final String CHECKBOX_SVG_PATH = "M5 13l4 4L19 7";
+  // Check Box
+  public static final String CHECKBOX_SVG_PATH =
+      "M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm0"
+          + " 16H5V5h14v14zM17.99 9l-1.41-1.42-6.59 6.59-2.58-2.57-1.42 1.41 4 3.99z";
+  // Payments
   public static final String CURRENCY_SVG_PATH =
-      "M12 8c-1.657 0-3 .895-3 2s1.343 2 3 2 3 .895 3"
-          + " 2-1.343 2-3 2m0-8c1.11 0 2.08.402 2.599 1M12 8V7m0 1v8m0 0v1m0-1c-1.11"
-          + " 0-2.08-.402-2.599-1M21 12a9 9 0 11-18 0 9 9 0 0118 0z";
+      "M19,14V6c0-1.1-0.9-2-2-2H3C1.9,4,1,4.9,1,6v8c0,1.1,0.9,2,2,2h14C18.1,16,19,15.1,19,14z"
```

Comment:
Per the discussion on https://github.com/seattle-uat/civiform/issues/1904 I think we want to keep the $ icon for currency

Reply:
Sure sounds good.